lib/config/presets/internal/monorepo.ts

```diff
@@ -30,35 +30,34 @@ const repoGroups = {
   'date-io': 'https://github.com/dmtrKovalenko/date-io',
   'devextreme-reactive': 'https://github.com/DevExpress/devextreme-reactive',
   'electron-forge': 'https://github.com/electron-userland/electron-forge',
-  'feathers-databases': 'https://github.com/feathersjs/databases',
   'ember-decorators': 'https://github.com/ember-decorators/ember-decorators',
   'graphql-modules': 'https://github.com/Urigo/graphql-modules',
   'ionic-native': 'https://github.com/ionic-team/ionic-native',
   'mdc-react': 'material-components/material-components-web-react',
   'ngx-formly': 'https://github.com/ngx-formly/ngx-formly',
   'ngxs-store': 'https://github.com/ngxs/store',
   'reach-ui': 'https://github.com/reach/reach-ui',
-  'react-apollo': 'https://github.com/apollographql/react-apollo',
+  'react-apollo': 'https://github.com/apollographql/react-apollo', // Deprecated by authors, do we want to keep it?
   'react-dnd': 'https://github.com/react-dnd/react-dnd',
   'react-navigation': 'https://github.com/react-navigation/react-navigation',
   'reactivestack-cookies': 'https://github.com/reactivestack/cookies',
   'reg-suit': 'https://github.com/reg-viz/reg-suit',
   'semantic-release': 'https://github.com/semantic-release/',
-  'shopify-app-bridge': 'https://github.com/Shopify/app-bridge',
-  'System.IO.Abstractions':
+  'system.io.abstractions':
     'https://github.com/System-IO-Abstractions/System.IO.Abstractions/',
-  'telus-tds': 'https://github.com/telusdigital/tds',
+  'telus-tds-core': 'https://github.com/telus/tds-core', // Original URL redirects to tds-core
```

Comment:
```suggestion
  'telus-tds-core': 'https://github.com/telus/tds-core',
```

But do all packages point to the new URL already, or should we keep both?

Reply:
I don't know about that, probably should stay safe and keep both.